httpcallback/router.go

```diff
@@ -70,16 +70,11 @@ func Index(c *gin.Context) {
 
 var routes = Routes{
 	{
-		"Index",
-		"GET",
-		"/",
-		Index,
+		Name: "Index", Method: "GET",
+		Pattern: "/", HandlerFunc: Index,
 	},
-
 	{
-		"DataChangeNotificationToNF",
-		strings.ToUpper("Post"),
-		"/sdm-subscriptions",
-		HTTPDataChangeNotificationToNF,
+		Name: "DataChangeNotificationToNF", Method: strings.ToUpper("Post"),
```

Comment:
Why not just type `POST` as is done in the previous set when using `GET`?
```suggestion
		Name: "DataChangeNotificationToNF", Method: "POST",
```